### ISSUE TITLE
ERROR (theano.sandbox.cuda): Failed to compile cuda_ndarray.cu: 'utf-8' codec can't decode....

### ISSUE THREAD

**Issue by kwonmha** [Theano/Theano#5607]
I'm using Win10 64bit, Python 3.5 on Anaconda3, CUDA 8.0, theano 0.8.2, VS2015 community
I already saw issue reporting same bugs I had but solutions suggested on the issue didn't help.
It was adding 'gbk' in decode function.
https://github.com/Theano/Theano/issues/3551

After I tried it, I got this message :
ERROR (theano.sandbox.cuda): Failed to compile cuda_ndarray.cu: 'cp949' codec can't encode character '\u80f6' in position 265: illegal multibyte sequence.

Thanks for any help.

**Comment by kwonmha** [comment]
Found solution.
Instead 'gbk', as I'm Korean, entered 'euc-kr'.

**Comment by nouiz** [comment]
Did you need to modify Theano to get it to work? If so, can you give us the diff? What is your local language of the OS?

You should not need to modify Theano to have it to work, so if that was the case, we should try to fix it upstream.

**Comment by kwonmha** [comment]
Well... I deleted the 'euc-kr' in theano/compat/init.py to roll back when the problem occured.
(After adding 'euc-kr', one more thing I did was removing " in compiler_bindir="C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin" and then it worked...)
So I added " in .theanorc.txt again.
And it got more clear(lots of warnings disappeared) and worked well.
I don't know why but today I have no error with importing theano.

But when the problem occured, I guess it was something about showing error message.
Because after I added 'euc-kr', I got error message in Korean unlike before.

Now it works well without any modification on library.
Don't know why.
